firestore.rules

```diff
@@ -1,9 +1,24 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
-    match /{document=**} {
-      allow read, write: if
-          request.time < timestamp.date(2022, 10, 20);
+    match /lobbies/{code} {
+      allow get: if true;
+
+      allow write: if true; // TODO: make an actual rule
+
+      match /privatePlayers/{uid} {
+        allow get: if request.auth.uid == uid;
+      }
+
+      match /promptAnswers/{uid} {
+        function validAnswer() {
+          return request.resource.data.answer is string
+            && 0 < request.resource.data.answer.size() && request.resource.data.answer.size() <= 50
+            && request.resource.data.answer == request.resource.data.answer.trim();
```

Comment:
shouldn't you trim before you check the size of the string? Example: '   ' is technically length 3 but once trimmed is 0

Reply:
i had the 3 space in between the '' github trimmed automatically

Reply:
I am checking that the answer is equal to the trimmed answer so it doesn't really matter